config/opensearch_dashboards.yml

```diff
@@ -238,5 +238,30 @@
 #data_source.encryption.wrappingKeyNamespace: 'changeme'
 #data_source.encryption.wrappingKey: [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
 
+#data_source.endpointDeniedIPs: [
+#    '127.0.0.0/8',
```

Comment:
Where are we getting this list from?

Reply:
These values users are free to change by themselves, most of them are common private IPs to block.